### DISCUSSION TITLE
LF3 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#142] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF3.html

Author: @AbbyANoble  
Reviewer: @tdegeorge

**Comment by AbbyANoble** [comment]
@tdegeorge - I've submitted the pull request (#163) for this one, so it should be available for review soon! There are two more activities I need to write, and I'll try to get those included soon. But it's mostly done.

**Comment by tdegeorge** [comment]
In Activity 3.3.3: I feel like part (c) is out of place....maybe put it at the end after they do the example with the table?

**Comment by tdegeorge** [comment]
Include the answer to Activity 3.3.8 (for instructor version)

**Comment by tdegeorge** [comment]
I think it looks good so far! When you are done with the last two activities, let me know and I will review those too.

**Reply by AbbyANoble** [reply]
Thanks! Made updates in #184 and added those last two activities in. CodeChat was being slow with all the graphs, so there's a good chance there's a mistake in those since they were hard to preview. 

The ones that are supposed to have vertical lines are wrong for sure. I didn't know how to do those, but I opened an issue (#185) for them.

**Comment by tdegeorge** [comment]
I think the rest looks good!

I noticed that you started putting in the graphs for the answers....do we want to do that throughout the section (like Activity 3.3.3 (parts a - c) and Activity 3.3.9 (part c))?

**Reply by AbbyANoble** [reply]
Not sure why I just stopped writing answers! haha. Thanks! Updated in #208
